topi/tests/python/test_topi_transform.py

```diff
@@ -322,3 +354,4 @@ def test_take():
     test_flip()
     test_expand_like()
     test_take()
+    test_stridedslice()
```

Comment:
No need to add a similar test to topi/tests/python_cpp/test_topi_transform.py?

Reply:
@kazum, the test/python_cpp will be removed in future as per  @tqchen , so i feel no need to add.
Anyway my python compute redirects to cpp implementation.